lib/services.template.ejs

```diff
@@ -3,6 +3,15 @@
 var urlBase = <%-: urlBase | q %>;
 var authHeader = 'authorization';
 
+function getHost(url) {
+  var m = url.match(/^(?:https?:)?\/\/([^\/]+)/);
+  if (m) {
+    return m[1]
+  }
+  return null;
```

Comment:
Actually, this can be probably simplified to

```js
return m ? m[1] : null;
```

Not a big deal though. You can either change this while rebasing, or ignore my comment (including the one above) and keep this as it is.